CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing Guidelines - Rocket.Chat.Java.SDK
+
+Great to have you here! Here are a few ways you can help make this project better!
+
+## Reporting an Issue
+
+[Github Issues](https://github.com/RocketChat/Rocket.Chat.Android/issues) are used to track todos, bugs, feature requests, and more.
+
+## Setting up a development environment
+In case you're interested in playing around with the code or giving something back, here are some instructions on how to set up your project:
+
+### Pre-requisites
+1. A Java Integrated Development Environment (e.g. IntelliJ IDEA, Eclipse IDE for Java Developers or NetBeans IDE).
+2. Clone this repo:
+```
+git clone https://github.com/RocketChat/Rocket.Chat.Java.SDK
+```
+3. Build the project.
+
+### Code style guide
+
+Before submitting a PR you should follow our [Coding Style](https://github.com/RocketChat/Rocket.Chat.Java.SDK/blob/develop/CODING_STYLE.md).
```

Comment:
You need to add the coding style file, or point to the one on the Android repository

Reply:
Blocked from #40

Reply:
@filipedelimabrito can you just point to https://github.com/RocketChat/java-code-styles/blob/master/CODING_STYLE.md